app/javascript/angular/code/services/graph.js

```diff
@@ -108,6 +108,7 @@ const afterSetExtremes = ({ streamId, times, showStatsCallback }) => e => {
         measurements,
         times
       });
+      measurementsByTime = dataByTime;
```

Comment:
this is something I removed in some of my previous changes and I just realized it might be used somewhere so I'm putting it back :D